docs/_tooling/README.md

```diff
@@ -0,0 +1,171 @@
+# Score Project Tooling Development Guide
+
+*This document is meant for *developers* of the `_tooling` aspect of the score repository.*  
+It should be treated as a 'get-started' guide, giving you all needed information to get up and running.
+
+## Quick Start
+
+1. Install Bazelisk (version manager for Bazel)
+2. Clone the repository  
+3. Setup the environment
+- *No Devcontainer*
+    1. Create the Python virtual environment:
+   ```bash
+   bazel run //docs:ide_support
+   ```
+    2. Select `.venv_docs/bin/python` as the python interpreter inside your IDE  
+    *Note: This virtual environment does **not** have pip, therefore `pip install` is not available.*  
+<br>
+
+- *With Devcontainer (VSCode)*
+    1. Click the `reopen current folder in dev container` prompt
+        -  If no prompt appears: `ctrl+shift+p` => `Dev Containers: Reopen in Containers`
+
+
+## Development Environment Requirements
+
+- **Operating System**: Linux (required)
+- **Core Tools**:
+  - Bazel 7.4.0 (via Bazelisk)
+  - Python 3.12
+  - Git
+
+
+### Key external tools used inside `_tooling`
+
+1. **Bazel Build System**
+   - Version: 7.4.0
+   - Primary build orchestrator
+   - Handles dependency management
+   - Coordinates testing and documentation
+   - Manages multi-repository setup
+
+2. **Documentation Tools**
+   - Sphinx with custom extensions
+   - Esbonio 0.16.5 for IDE integration
+   - Real-time documentation validation
+
+3. **Development Tools**
+   - Gitlint for commit message standards
+   - Pytest for testing infrastructure
+   - Custom formatters and linters
+
+
+
+## Tooling Directory Architecture
+
+```
+docs/_tooling/
+├── assets/           # Documentation styling (CSS)
+├── conf_extras/      # Sphinx configuration extensions
+├── decision_records/ # Architecture Decision Records (ADRs)
+├── extensions/       # Custom Sphinx extensions
+│   └── score_metamodel/
+│       ├── checks/  # Sphinx-needs validation
+│       └── tests/   # Extension test suite
+└── templates/        # Documentation templates
+```
+
+## Development Workflow
+
+
+#### Documentation Development
+```bash
+# Incremental documentation build (recommended during development)
+bazel run //docs:incremental
+
+# Full documentation build
+bazel build //docs:docs
+```
+
+#### Testing
```

Comment:
The IDE integration is missing. I feel this is more important than command line commands.